cattle/plugins/docker/compute.py

```diff
@@ -42,8 +43,13 @@
     ('capDrop', 'cap_drop'),
     ('dnsSearch', 'dns_search'),
     ('dns', 'dns'),
+    ('extraHosts', 'extra_hosts'),
     ('publishAllPorts', 'publish_all_ports'),
     ('lxcConf', 'lxc_conf'),
+    ('logConfig', 'log_config'),
+    ('securityOpt', 'security_opt'),
+    ('restartPolicy', 'restart_policy'),
```

Comment:
This doesn't present a capitalization issue around maximumRetryCount != MaximumRetryCount ?